synapseutils/walk.py

```diff
@@ -11,6 +26,9 @@ def walk(syn, synId):
 
     :param synId:          A synapse ID of a folder or project
 
+    :param includeTypes:   Must be a list of entity types (ie. ["folder","file"]) which can be found here:
+                           http://docs.synapse.org/rest/org/sagebionetworks/repo/model/EntityType.html
+
     Example::
```

Comment:
Ya.  Basically people can choose what they want returned.  So if they only want files and nothing else they would just specify ["file"] - although that may break the function as it needs folders to generate the dirpath and dirname.  Maybe Ill have to specify that you must include ['folder','file'] at the minimum?